File: examples/python/documentation/my_constructs.py
from constructs import Construct


class KubernetesWebAppDeployment(Construct):
    def __init__(self, scope, name, **attributes):
        super().__init__(scope, name)

        # The content is left to the readers imagination

class DockerBackend(Construct):
    def __init__(self, scope: Construct, id: str, vpc_id: str, docker_image: str):
        super().__init__(scope, id)

File: examples/csharp/documentation/MyConstructs.cs
// Copyright (c) HashiCorp, Inc.
// SPDX-License-Identifier: MPL-2.0

using Constructs;
using kubernetes;
using System.Collections.Generic;

namespace MyConstructs
{
    class KubernetesWebAppDeployment : Construct
    {
        public KubernetesWebAppDeployment(Construct scope, string id, Dictionary<string, object> config) : base(scope, id)
        {
            // Content is left to the readers imagination
        }
    }
}